library/redis

```diff
@@ -2,14 +2,25 @@
 
 Maintainers: Tianon Gravi <admwiggin@gmail.com> (@tianon),
              Joseph Ferguson <yosifkit@gmail.com> (@yosifkit)
-GitRepo: https://github.com/docker-library/redis.git
+             David Maier <david.maier@redis.com> (@dmaier-redislabs)
+GitRepo: https://github.com/redis/docker-library-redis.git
 
-Tags: 7.4-rc1, 7.4-rc, 7.4-rc1-bookworm, 7.4-rc-bookworm
+Tags: 7.4-rc2, 7.4-rc, 7.4-rc2-bookworm, 7.4-rc-bookworm
+Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, mips64le, ppc64le, s390x
+GitCommit: c828d8c6d982f0d45e1e8f564aae50b42f6d1ec0
+Directory: 7.4-rc/debian
+
+Tags: 7.4-rc2-alpine, 7.4-rc-alpine, 7.4-rc2-alpine3.20, 7.4-rc-alpine3.20
+Architectures: amd64, arm32v6, arm32v7, arm64v8, i386, ppc64le, riscv64, s390x
+GitCommit: c828d8c6d982f0d45e1e8f564aae50b42f6d1ec0
+Directory: 7.4-rc/alpine
+
+Tags: 7.4-rc1, 7.4-rc1-bookworm
 Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, mips64le, ppc64le, s390x
 GitCommit: 0d682fed252b85f39d2033294eab217be02f95a1
 Directory: 7.4-rc/debian
 
-Tags: 7.4-rc1-alpine, 7.4-rc-alpine, 7.4-rc1-alpine3.20, 7.4-rc-alpine3.20
+Tags: 7.4-rc1-alpine, 7.4-rc1-alpine3.20
```

Comment:
These `rc1` versions should be removed with `rc2`; removing tags here will remove them from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding them on the official build servers), but the tags will still be available to users who want them.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

Reply:
OK, this should be fixed now. Part of the issue on my side yesterday night was that the `generate-stackbrew-library` script didn't execute without errors in my development environment. So I just went ahead and generated the entries manually. I fixed the issue now and the new output should be what you expect from the script.